manager/mocks/channels.go

```diff
@@ -29,7 +28,7 @@ func (crm *channelRepositoryMock) Save(channel manager.Channel) (string, error)
 	defer crm.mu.Unlock()
 
 	crm.counter += 1
-	channel.ID = strconv.Itoa(crm.counter)
+	channel.ID = fmt.Sprintf("123e4567-e89b-12d3-a456-%012d", crm.counter)
```

Comment:
Theoretically, this will start giving invalid UUIDs after 9 entities. 

Instead, let's be more explicit about the way we want to use this implementation. Predefine an array of n valid uuids, and cycle through it using the counter modulo n as an index. Yes, there will be duplicates, but this should be a stub anyhow, and later on we can improve its logic if it is necessary.

Reply:
Please ignore this comment. I misunderstood your implementation.